src/main/java/bio/terra/workspace/app/configuration/spring/BeanConfig.java

```diff
@@ -15,6 +19,21 @@
 
 @Configuration
 public class BeanConfig {
+  @Bean
+  public Tracer tracer() {
+    return Tracing.getTracer();
+  }
+
+  @Bean
+  public TracingAsyncClientHttpRequestInterceptor requestInterceptor() {
```

Comment:
did we ever figure out why we needed this? (/sanity check it's still needed?)
maybe a comment about why we have it would be helpful. probably ditto for the other beans.

Reply:
We never did, it still is.
Added comments for the beans.